### ISSUE TITLE
HorizontalScrollSnap.cs misspelling

### ISSUE THREAD

**Issue by SimonDarksideJ** [Unity-UI-Extensions/com.unity.uiextensions#352]
<sup>Issue created by Michael Pizik as [Bitbucket Issue #&#8203;213](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/213) on 2018.02.19 11:58.</sup>
In line 52 code:

```C#

 if (_scroll_rect.velocity.x > 0.01 || _scroll_rect.velocity.x < 0.01)
```
should be

```C#
 if (_scroll_rect.velocity.x > 0.01 || _scroll_rect.velocity.x < -0.01)

```

Look at VerticalScrollSnap.cs line 52

**Comment by SimonDarksideJ** [comment]
<sup>On [2018.02.19 15:23](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/213#comment-43389490), @SimonDarksideJ commented:</sup>
HorizontalScrollSnap.cs updated to match VerticalScrollSnap - Resolves #213

Referenced by 4d913f33a88e

**Comment by SimonDarksideJ** [comment]
<sup>On 2018.02.19 15:23 @SimonDarksideJ modified issue:</sup>
**status** changed `new` → `resolved`
